posts/nataliametz/Data-Driven-Design-Making-Smarter-Decisions-for-User-Satisfaction.md

```diff
@@ -0,0 +1,29 @@
+---
+title: 'Data-Driven Design: Making Smarter Decisions for User Satisfaction' 
+published: false
+tags: 'ux, ux-practices, ux-design, decision-making, optimization'
```

Comment:
Solo 4 tags son permitidos por dev.to, y los guiones me parece no estan permitidos, habria que removerlos